include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -161,10 +161,10 @@ struct __get_first_range_type
 };
 
 template <typename _Range, typename... _Ranges>
-constexpr _Range
-__get_first_range(_Range&& __rng, _Ranges&&...)
+auto
+__get_first_range_size(_Range&& __rng, _Ranges&&...)
```

Comment:
As far as I remember we agreed on `const Range&`. @MikeDvorskiy please correct me if I am wrong

Reply:
As I remember, we agreed to use `const Range&` to avoid warning -Wreturn_std_move when return first range. Now I don't see the reason to change it to `const Range&`, because I don't see the difference, we don't return __rng now. Please correct me if I am wrong.

Reply:
yes, `const Range&`